src/lib/CMakeLists.txt

```diff
@@ -126,7 +126,7 @@ if(CMAKE_APP_TYPE STREQUAL "BMDK" OR
    CMAKE_APP_TYPE STREQUAL "bristleback" OR
    CMAKE_APP_TYPE STREQUAL "rs232_expander" OR
    APP_NAME STREQUAL "bringup_mote" OR
-   APP_NAME STREQUAL "mote_bristlefin" OR
+   APP_NAME MATCHES "mote_bristlefin" OR
```

Comment:
This enabled the debug build, where `-dbg` gets appended to the app name, to also add bristlefin.cpp to `LIB_FILES`.